[SPMD][BE] Remove the legacy tracing code

fegin opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #101012
* __->__ #100858

Remove the legacy tracing code as it cause several test and benchmark issues.

Differential Revision: [D45649123](https://our.internmc.facebook.com/intern/diff/D45649123/)

fegin committed:
[SPMD] Remove the legacy tracing code

Remove the legacy tracing code as it cause several test and benchmark issues.

Differential Revision: [D45649123](https://our.internmc.facebook.com/intern/diff/D45649123/)

[ghstack-poisoned]

fegin committed:
Update on "[SPMD] Remove the legacy tracing code"


Remove the legacy tracing code as it cause several test and benchmark issues.

Differential Revision: [D45649123](https://our.internmc.facebook.com/intern/diff/D45649123/)

[ghstack-poisoned]

fegin committed:
Update on "[SPMD] Remove the legacy tracing code"


Remove the legacy tracing code as it cause several test and benchmark issues.

Differential Revision: [D45649123](https://our.internmc.facebook.com/intern/diff/D45649123/)

[ghstack-poisoned]

fegin committed:
Update on "[SPMD] Remove the legacy tracing code"


Remove the legacy tracing code as it cause several test and benchmark issues.

Differential Revision: [D45649123](https://our.internmc.facebook.com/intern/diff/D45649123/)

[ghstack-poisoned]

wanchaol reviewed (approved):
Nice, thanks for cleaning this up!

fegin committed:
Update on "[SPMD][BE] Remove the legacy tracing code"


Remove the legacy tracing code as it cause several test and benchmark issues.

Differential Revision: [D45649123](https://our.internmc.facebook.com/intern/diff/D45649123/)

[ghstack-poisoned]

fegin committed:
Update on "[SPMD][BE] Remove the legacy tracing code"


Remove the legacy tracing code as it cause several test and benchmark issues.

Differential Revision: [D45649123](https://our.internmc.facebook.com/intern/diff/D45649123/)

[ghstack-poisoned]

fegin commented:
@pytorchbot merge

fegin committed:
Update on "[SPMD][BE] Remove the legacy tracing code"


Remove the legacy tracing code as it cause several test and benchmark issues.

Differential Revision: [D45649123](https://our.internmc.facebook.com/intern/diff/D45649123/)

[ghstack-poisoned]

fegin commented:
@pytorchbot merge

fegin commented:
@pytorchbot merge

fegin committed:
Update on "[SPMD][BE] Remove the legacy tracing code"


Remove the legacy tracing code as it cause several test and benchmark issues.

Differential Revision: [D45649123](https://our.internmc.facebook.com/intern/diff/D45649123/)

[ghstack-poisoned]
